app/api/controllers/RequestHandler.scala

```diff
@@ -123,21 +126,34 @@ object RequestHandler {
 
       implicit class Response(result: Result) {
 
+        private val HttpDateFormatter = DateTimeFormatter
+          .ofPattern("EEE, dd MMM yyyy HH:mm:ss z", Locale.ENGLISH)
+          .withZone(ZoneId.of("GMT"))
+
         def withApiHeaders(correlationId: String, responseHeaders: (String, String)*)(implicit appConfig: AppConfig, apiVersion: Version): Result = {
           val maybeDeprecatedHeader =
```

Comment:
tbh this all ought to go into a separate class or object with its own unit test. RequestHandler is already pretty scary and has multiple personalities.

Also to emphasise, needs a unit test!